### PR TITLE
Add: Policy section for content creators

### PR DESCRIPTION
**Pull request by nielsmh**
Just being explicit about what the license says and that the intention is indeed what it says.

### CODE REVIEW

File: pages/policy.html [hunk 1]
```diff
@@ -5,6 +5,22 @@
 ---
 
 <div id="section-full">
+    <div class="section-header">
+        <h3>Gameplay streaming, Let's Play content, and Monetization Statement</h3>
+    </div>
+    <div class="section-item">
+        <div class="content">
+            <p>The core game of OpenTTD is released under the GNU General Public License (GPL) version 2, as are the OpenGFX, OpenSFX, and OpenMSX basetset content packs. The GPL permits use of the software for any purpose including commercial purposes.</p>
```
Comment (TrueBrain) [inline pages/policy.html:13]: `basetset`, too many `t`s :)

File: pages/policy.html [hunk 2]
```diff
@@ -5,6 +5,22 @@
...
+            <h3>General permission for the unmodified game</h3>
+            <p>It is the intention of the development teams behind OpenTTD and the open baseset content packs that any Let's Play, review, and other similar content, is permitted with no limitations. Monetization and commercial use of the content is permitted without limitations.</p>
+            <h3>Non-coverage of use of other baseset data</h3>
+            <p>Note that the above statement of general permission only applies to using the OpenTTD core game with the OpenGFX graphics baseset, and optionally the OpenSFX sound effects baseset, and optionally OpenMSX music baseset.</p>
```
Comment (TrueBrain) [inline pages/policy.html:17]: twice the `optionally` is a bit weird. Maybe just put OpenSFX / OpenMSX together?

File: pages/policy.html [hunk 3]
```diff
@@ -5,6 +5,24 @@
...
+        <div class="content">
+            <p>The core game of OpenTTD is released under the GNU General Public License (GPL) version 2, as are the OpenGFX, OpenSFX, and OpenMSX baseset content packs. The GPL permits use of the software for any purpose including commercial purposes.</p>
+            <p>Following is a statement of intention from the developers. This is not a legal statement. If any questions arise from this statement, please consult the license text of the involved software and game data.</p>
+            <h3>General permission for the unmodified game</h3>
+            <p>It is the intention of the development teams behind OpenTTD and the open baseset content packs that any Let's Play, review, and other similar content, is permitted with no limitations. Monetization and commercial use of the content is permitted without limitations.</p>
+            <p>Note that this statement does not imply any endorsement from any of the developers of OpenTTD or the baseset content towards any content creators. Any views expressed by content creators in content using the OpenTTD game and baseset content packs are purely those of the content creator.</p>
```
Comment (TrueBrain) [inline pages/policy.html:17]: This sentence is a bit hard to read. It says `content` a lot, but not always in the same context. And in general it feels a bit meh vs the other sentences. But I am also having a hard time coming up with an alternative, so I am not really useful here, sorry :(

**Comment by TrueBrain** [comment]
Currently this page is called "Privacy Policy"; with this addition, that is no longer true.

Browsing some other sites, I see others solve that with for example:
- Make a "Content Creator Guideline" page for this part specific
- Rename this page to "Legal"

I can help out with the first if that is what we want. The second is relative easy :)

File: pages/policy.html (hunk 3 above)
Comment (nielsmh) [reply pages/policy.html]: I changed it to "game data" when that's what it means.

File: pages/policy.html [hunk 4]
```diff
@@ -5,6 +5,29 @@
...
+            <p>Following is a statement of intention from the developers. This is not a legal statement. If any questions arise from this statement, please consult the license text of the involved software and game data.</p>
+            <h3>General permission for the unmodified game</h3>
+            <p>It is the intention of the development teams behind OpenTTD and the open baseset game data packs that any Let's Play, review, and other similar content, is permitted with no limitations. Monetization and commercial use of the game and game data is permitted without limitations.</p>
+            <p>Note that regardless of this statement, any distribution of the game and game data files still falls under the restrictions of the respective licenses.</p>
+            <p>Note that this statement does not imply any endorsement from any of the developers of OpenTTD or the baseset game data towards any content creators. Any views expressed by content creators in content using the OpenTTD game and baseset game data packs are purely those of the content creator.</p>
+            <h3>Non-coverage of use of other baseset data</h3>
```
Comment (TrueBrain) [inline pages/policy.html:23]: Nitpick: `of use of other` -> `of use with` (similar as with `add-on game data`. `of .. of` reads weird :)

**Review by frosch123 — APPROVED** [review]

**Pull request merged**